### PR TITLE
fix Win32ApiTests against x86 platform

### PR DESCRIPTION
**Pull request by NikolayXHD**
Fix faling Win32ApiTests tests when run against x86 platform, issue was discovered
and reported https://github.com/gitextensions/gitextensions/pull/6111#issuecomment-456175161 by @vbjay 

## Test environment(s)

- Windows 7 SP1

Win32Api tests were successfully run against x86 and x64 platforms

### CODE REVIEW

**Comment by RussKie** [comment]
@mstv could you please apply the fix locally and confirm the fix?

**Comment by amaiorano** [comment]
Checked out @NikolayXHD's branch locally, built and ran the Win32ApiTests and it passed. However, I am on Windows 10 64-bit, so I cannot test on 32-bit platform.

**Comment by NikolayXHD** [comment]
I was changing the target platform of Test project to x86, it made difference - before the fix, tests were failing only with target platform x86

**Comment by vbjay** [comment]
I am on a 64 bit box.

**Comment by amaiorano** [comment]
> I was changing the target platform of Test project to x86, it made difference - before the fix, tests were failing only with target platform x86

Ah, right. I'll test that as well ASAP.

File: UnitTests/GitExtUtils/Win32ApiTests.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class Win32ApiTests
 
         private static IntPtr ToLParam(short x, short y) =>
             new IntPtr(
-                ((long)x & 0x00000000_0000FFFF) |
-                (((long)y & 0x00000000_0000FFFF) << 16));
+                x & 0x0000_FFFF |
+                (y & 0x0000_FFFF) << 16);
```
[the remark this answers is not in the document]
Comment (NikolayXHD) [reply UnitTests/GitExtUtils/Win32ApiTests.cs]: `short` `y` is converted to `int` before applying `&`

**Comment by NikolayXHD** [comment]
> @NikolayXHD could you please rebase on top of latest master and add your sign off under Git Extensions Project Contributors Certification of Origin and Rights as part of this PR.
> Thank you.

Oops I did that in another pending PR, is it ok?

**Pull request merged**